src/main/java/de/tarent/entities/ParticipantFeedback.java

```diff
@@ -0,0 +1,31 @@
+package de.tarent.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Entity
+@EqualsAndHashCode
+@ToString
+@JsonIgnoreProperties(value = {"id"}, ignoreUnknown = true)
+public class ParticipantFeedback extends PanacheEntity {
+
+    @Length(max = 255)
```

Comment:
`@Length` does not imply `@NotNull`

Reply:
please check for this error in the whole document

Reply:
I think `@NotNull` is missing because the field is not required